ui/open.c

```diff
@@ -17,7 +17,9 @@ int main(int argc, char **argv) {
     FILE *sm_file;
 
     xmss_params params;
-    uint32_t oid;
+    uint32_t oid = 0;
+    u_int8_t buffer[XMSS_OID_LEN];
```

Comment:
Please use `uint8_t` (or for consistency with the majority of this codebase: `unsigned char`). This applies to the other uses of `u_int8_t` as well.